package.json

```diff
@@ -38,9 +36,8 @@
   "scripts": {
     "docs": "jsdoc -c .jsdoc.js",
     "generate-scaffolding": "repo-tools generate all --except lib_readme",
-    "lint": "eslint samples/",
+    "lint": "gts check && eslint '**/*.js'",
```

Comment:
Would this catch js files in `build/`?

Reply:
Nope! `build/` is in the default `.eslintignore` :)